src/utils/getFormatter.js

```diff
@@ -12,9 +12,9 @@ const NATIVE_FORMATTER_PATH = path.join('src', 'formatters');
  */
 const loadFormatter = function(formatterPath) {
   try {
-    /* eslint-disable import/no-dynamic-require */
+    // eslint-disable import/no-dynamic-require
```

Comment:
`// eslint-disable-next-line  import/no-dynamic-require` allows you to delete the whole `// eslint-enable` line